File: Patterns/4. Right Triangle Patterns (Top-Right)/40. Right Triangle TR Continuous Numbers/trcn.py
n = int(input("Enter a Number : "))
num=1
for i in range(1,n+1):
    # print spaces
    for j in range(1,n-i+1):
        print(" ",end = "  ")
    # print stars
    for j in range(1,i+1):
        print(f"{num:3}",end = '')
        num+=1
    print()

'''

Output:

              1
           2  3
        4  5  6
     7  8  9 10
 11 12 13 14 15  
  
'''

File: Patterns/04. Right Triangle Patterns (Top-Right)/40. Right Triangle TR Continuous Numbers/trcn.cpp
#include <iostream>
#include <iomanip>
using namespace std;
int main(){
    int n,num=1;
    cout<<"Enter a Number : ";
    cin>>n;

    for(int i=1; i<=n; i++){
        // print spaces
        for(int j=1; j<=n-i; j++){
            cout<< "   ";
        }
        // print stars
        for(int j=1; j<=i;j++){
            cout<<setw(3)<<num;
            num++;
        }
        cout << '\n';
    }
}

/* 

Output

              1
           2  3
        4  5  6
     7  8  9 10
 11 12 13 14 15    

*/ 